Website/src/items/templates/edgewebsite/cmp entities/Vendor/Data/Level.yml

```diff
@@ -2,11 +2,11 @@
 ID: "36288362-9494-4174-a8a2-d846d958149d"
 Parent: "3715f124-6d1e-4ad0-9b80-0e905fabd66e"
 Template: "455a3e98-a627-4b40-8035-e683a0331ac7"
-Path: /sitecore/templates/Project/edgewebsite/cmp entities/Vendor/Data/Level
+Path: /sitecore/templates/Project/edgewebsite/CMP Entities/Vendor/Data/Level
 SharedFields:
 - ID: "1eb8ae32-e190-44a6-968d-ed904c794ebf"
   Hint: Source
-  Value: /sitecore/content/CMP/Vendor Levels
+  Value: /sitecore/content/EdgeWebsite/Data/CMP Data/Vendor Levels
```

Comment:
Please check the "Broken links report" for the master database and fix all the broken links. For example, this field source was updated but the items from that template have a hardcoded string value in the field instead of an item reference. This is caused by data coming from ContentHub. If we do not expect item references in this field, it would be wise to remove the "source" value in the template.

Reply:
OK, broken link report is now clean (except for droplist fields which are appearing by mistake)